applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -470,8 +491,12 @@ class ExplicitCentralDifferencesScheme
     /**
      * @brief This method updates the translation DoF
      * @param itCurrentNode The iterator of the current node
+     * @param DoF The current dimention of the problem
      */
-    void UpdateTranslationalDegreesOfFreedom(NodeIterator itCurrentNode)
+    void UpdateTranslationalDegreesOfFreedom(
+        NodeIterator itCurrentNode,
+        const SizeType DoF = 3
```

Comment:
same

Reply:
This is for each node, better to provide externally to avoid calling the model part everytime

Reply:
still...bad name for the variable